src/NUnitFramework/tests/TestUtilities/CallbackEventHandler.cs

```diff
@@ -0,0 +1,40 @@
+// ***********************************************************************
+// Copyright (c) 2014 Charlie Poole, Rob Prouse
```

Comment:
2018

Reply:
This code was just moved, so I took the copyright date from the original file. 😊